### DISCUSSION TITLE
External tooltip library?

### DISCUSSION THREAD

**Discussion by harshad1** [gsantner/markor#1830] [General] [answered]
@gsantner You were right, the android tooltip system has very low flexibility and is a pain to use. 

What do you think about using an external tooltip library like https://github.com/skydoves/Balloon (this one is nice but may be a little heavyweight)?

**Comment by gsantner** [comment, marked as answer]
Not fond of it. I would rather drop it then add a dep. Not sure how much code it is, but new textview in code, setText, setPos(centerScreen), setOpacity, setBackground. Anyway it is probably always disturbing down there.
